src/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -60,6 +60,7 @@ const StoragePoolTypeRow = ({ onValueChanged, dialogValues, libvirtVersion, pool
         { type: 'iscsi', detail: _("iSCSI target") },
         { type: 'disk', detail: _("Physical disk device") },
         { type: 'logical', detail: _("LVM volume group") },
+        { type: 'fs', detail: _("Pre-formatted Block Device") },
```

Comment:
This is a rather curious name -- could this be "Directory"? Does libvirt actually care/check if this is a full block device, or could it just be any directory? But then again, we already have "Filesystem directory" -- how does this  differ from `dir`?

Is there some reference in the libvirt documentation how it describes these types?

Reply:
Also, there is "disk" type already. Between "dir" and "disk" there isn't a lot of room, so I'm interested in when users would pick that over the other two.

Reply:
> This is a rather curious name -- could this be "Directory"?

'fs' functions differently from 'dir'.

'dir' registers the storage pool under the specified directory.
'disk' registers the entire specified disk as a storage pool.
'fs' registers the specified formatted partition as a storage pool.

The libvirt documentation describes fs as follows:
- https://libvirt.org/storage.html#filesystem-pool

Reply:
Right, I know the syntactics difference -- I mean, what's the use case for this? when is "fs" better than "dir" or "disk"?

Reply:
If you pass 'disk' to a guest, an error may occur when the host detects the guest's disk.
By using 'fs', you can avoid problems with the host grabbing the disk for the guest to use.

The RHEL documentation says to assign guests at the partition level instead of the entire disk to avoid the above problems:
```
When whole disks or block devices are passed to the VM, the VM will likely partition it or create its own LVM groups on it. This can cause the host machine to detect these partitions or LVM groups and cause errors.
These errors can also occur when you manually create partitions or LVM groups and pass them to the VM.
To avoid theses errors, use file-based storage pools instead.
```
- https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_virtualization/managing-storage-for-virtual-machines_configuring-and-managing-virtualization#proc_creating-disk-based-storage-pools-using-the-web-console_assembly_managing-virtual-machine-storage-pools-using-the-web-console

Reply:
I see, thanks! That almost sounds like the "disk" should be deprecated in favor of "fs".. Thanks for the pointers! (Note, I am very much not a libvirt expert, so this is learning for me)